app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt

```diff
@@ -140,6 +143,11 @@ class BrowserWebViewClient(
     override fun onPageStarted(webView: WebView, url: String?, favicon: Bitmap?) {
         try {
             Timber.v("onPageStarted webViewUrl: ${webView.url} URL: $url")
+            url?.let {
+                GlobalScope.launch {
```

Comment:
For this one, we can inject @AppCoroutineScope private val appCoroutineScope: CoroutineScope

Reply:
👍